Branch-SDK/src/io/branch/referral/PrefHelper.java

```diff
@@ -1024,6 +1032,18 @@ public void disableSmartSession() {
         BNC_Smart_Session = false;
     }
 
+    public void setRequestMetadata(@NonNull String key, Object value) {
```

Comment:
No - I originally had it as JSONObject, but moved it to generic Object when I modified the exposed class in `Branch.java`. It allows for someone to do:

`addRequestToMetadata("key", "value")`

let me know thoughts